onnx/shape_inference/implementation.cc

```diff
@@ -404,9 +408,9 @@ class ShapeInferenceImplBase {
     // Resolve domain for node
     auto dit = opset_imports.find(n.domain());
     if (dit == opset_imports.end()) {
-      // Both "" and "ai.onnx" refer to the default ONNX domain
-      if (n.domain() == "") {
-        dit = opset_imports.find("ai.onnx");
+      // Both "" (ONNX_DOMAIN) and "ai.onnx" (AI_ONNX_DOMAIN) refer to the default ONNX domain
+      if (n.domain() == ONNX_DOMAIN) {
```

Comment:
What if there are other domains, like custom ones?

Reply:
It is handled 2 lines above (new line number 409). This is just a fallback, because there are two ways of referring to the standard domain. (Anyway, this is just pre-existing code, not part of the current PR ... I was just cleaning up the reference to the string constants using the symbolic identifier.